packages/beacon-node/test/e2e/sync/unknownBlockSync.test.ts

```diff
@@ -144,13 +145,15 @@ describe("sync / unknown block sync for fulu", () => {
       await connected;
       loggerNodeA.info("Node A connected to Node B");
 
-      const headInput = BlockInputPreData.createFromBlock({
-        block: head,
+      const headInput = BlockInputColumns.createFromBlock({
```

Comment:
Very nice catch.  Was looking for this bug and totally missed that this was the wrong type of blockInput.  Your rock sir!!